spec/controllers/post_spec.rb

```diff
@@ -4,8 +4,8 @@
   describe "GET #index" do
     it "responds successfully with an HTTP 200 status code" do
       get :index
+
       expect(response).to be_success
```

Comment:
Ah, I was thinking of :ok as equal to 200. https://github.com/rack/rack/blob/2f782ce099989e214466872c797f530b5349357d/lib/rack/utils.rb#L509